src/sql/parts/dashboard/widgets/insights/views/charts/chartInsight.component.ts

```diff
@@ -212,14 +213,14 @@ export abstract class ChartInsight extends Disposable implements IInsightsView {
 			if (this._config.labelFirstColumn) {
```

Comment:
There are other extended classes that override this, so probably a better solution is to filter the data when the data is set